src/__init__.py

```diff
@@ -0,0 +1,6 @@
+print('Hello, world!')
+name = input('What is your name?\n')
```

Comment:
It looks much more clear if you task description stands closely to task and 
separated from other tasks with two blank lines: 
```python
pass  # last line of previous task


# next task docstring
pass  # my next task first line  
```